packages/blade/src/components/Link/Link/Link.tsx

```diff
@@ -0,0 +1,87 @@
+import type { ReactElement, SyntheticEvent } from 'react';
+import type { IconComponent } from '../../Icons';
+import { BaseLink } from '../BaseLink';
+
+type LinkCommonProps = {
+  variant?: 'anchor' | 'button';
+  icon?: IconComponent;
+  iconPosition?: 'left' | 'right';
+  isDisabled?: boolean;
+  onClick?: (event: SyntheticEvent) => void;
+  href?: string;
+  target?: string;
+  accessibilityLabel?: string;
+};
+
+/*
+  Mandatory children prop when icon is not provided
+*/
+type LinkWithoutIconProps = LinkCommonProps & {
```

Comment:
aren't these props repeating in base as well as here?

Reply:
Yes. The same situation like BaseButton & Button. We could directly import the BaseLink props here and use them but that would lead to Link's props changing if something is changed in BaseLink. We had the same discussion on BaseButton as well and decided to keep them separate